src/openepd/model/specs/generated/aggregates.py

```diff
@@ -0,0 +1,69 @@
+#
+#  Copyright 2024 by C Change Labs Inc. www.c-change-labs.com
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#  This software was developed with support from the Skanska USA,
+#  Charles Pankow Foundation, Microsoft Sustainability Fund, Interface, MKA Foundation, and others.
+#  Find out more at www.BuildingTransparency.org
+#
+from openepd.compat.pydantic import pyd
+from openepd.model.base import BaseOpenEpdSchema
+from openepd.model.specs.base import BaseOpenEpdHierarchicalSpec
+from openepd.model.specs.generated.enums import AggregateGradation, AggregateWeightClassification
+from openepd.model.validation.numbers import RatioFloat
+from openepd.model.validation.quantity import LengthMmStr
+
+
+class AggregateApplication(BaseOpenEpdSchema):
+    """Application for aggregates."""
+
+    concrete: bool | None = pyd.Field(default=None, description="Aggregates used in concrete and masonry applications")
+    asphalt: bool | None = pyd.Field(
+        default=None, description="Aggregates used in bituminous paving and surface applications"
+    )
+    unbound: bool | None = pyd.Field(default=None)
+
+
+class AggregatesV1(BaseOpenEpdHierarchicalSpec):
+    """Aggregates performance specification."""
+
+    _EXT_VERSION = "1.0"
+    """Construction Aggregates, including sand, gravel, crushed stone, etc.
```

Comment:
I don't think that this docstring should belong to `_EXT_VERSION`. Please, check other places.

Reply:
Thanks, good point!